### PR TITLE
[HLX] ReGrowth - Mutated Animals Pack

### PR DESCRIPTION
**Pull request by Breadbox2k19**
## Additions

Patchy for a mod.

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [ ] Playtested a colony (specify how long)

### CODE REVIEW

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
In addition to a few specific changes to be made, more generally I'd also like you to tidy up the formatting a bit; make the alignment a bit more uniform, etc.

**Review by Breadbox2k19 — COMMENTED** [review]
Mod finalised, let me know if there are any errors on your end.

File: Patches/[HLX] ReGrowth - Mutated Animals Pack/Deffclaw.xml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+		</mods>
+
+		<match Class="PatchOperationSequence">
+			<operations>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/ThingDef[defName="RG-WF_WastelandDeathclaw"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Birdlike</bodyShape>
```
[the remark this answers is not in the document]
Comment (N7Huntsman) [reply Patches/[HLX] ReGrowth - Mutated Animals Pack/Deffclaw.xml:15]: I disagree. Quadruped *perhaps,* but Birdlike is intended for things with comparatively long, thin necks. necks, such as emu, ostriches. It's certainly not appropriate for a Deathclaw.

[the remark this answers is not in the document]
Comment (Breadbox2k19) [reply Patches/[HLX] ReGrowth - Mutated Animals Pack/Deffclaw.xml]: Doesn't birdlike simply meant height x0.75 and width x1 . Of course deathclaw doesn't have a long neck or anything, bird, like is simply meant to represent the fact that death claw doesn't stand up straight and is as wide as is it is tall.

**Review by N7Huntsman — APPROVED** [review]
Everything seems good, pending a quick look into an odd start-up error.

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**